app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -57,6 +67,42 @@ describe("PageEditAudience", () => {
     });
   });
 
+  it("redirects to the review page if the experiment status is review", async () => {
+    const { mock, experiment } = mockExperimentQuery("demo-slug", {
+      status: NimbusExperimentStatus.REVIEW,
+    });
+    render(<Subject mocks={[mock]} />);
+    await waitFor(() => {
+      expect(navigate).toHaveBeenCalledWith(
+        `${BASE_PATH}/${experiment.slug}/request-review`,
+      );
+    });
+  });
+
+  it("redirects to the design page if the experiment status is live", async () => {
```

Comment:
Probably unrelated, but, the Design page shows a loading spinner with at least live experiment since the visualization API request 500's 🤔 Try out `https://localhost/nimbus/robust-didactic-open-architecture/design` in your branch. Might need a follow up issue.

Reply:
Are you referring to this? https://github.com/mozilla/experimenter/pull/4233#issuecomment-743397223

Reply:
If so, for now I'd consider it an acceptable risk because there shouldn't be a non-addressable scenario where this is happening (see listed items). But I agree it could use a followup.

Reply:
Oh you might have been referring to something else, I'm noticing an indefinite loader when I hit /design with an Accepted experiment.

Reply:
I did fix the issue I was experiencing here, but going to leave this open in case you were talking about something else :)

Reply:
Yeahhh, I somehow lost my GCP credential file. 🤦‍♀️